lib/ecto/migration/runner.ex

```diff
@@ -42,11 +41,11 @@ defmodule Ecto.Migration.Runner do
   @doc """
   Starts the runner for the specified repo.
   """
-  def start_link(parent, repo, direction, migrator_direction, level) do
+  def start_link(parent, repo, direction, migrator_direction, level, log_sql) do
```

Comment:
Let's change the last argument to be `log` and contain a keyword `[level: level, sql: log_sql]`, I'm not a huge fan of multiple boolean flags passed around.